content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -477,34 +475,98 @@ validation and auto-completion.
 
 ## Top-level plot definitions
 
-The list of `plots` contains one or more user-defined `dvc plots` (paths
+The list of `plots` contains one or more user-defined top-level plots (paths
 relative to the location of `dvc.yaml`).
 
-This example makes output `auc.json` viable for visualization, configuring keys
-`fpr` and `tpr` as X and Y axis, respectively:
+Every plot has to have its own ID. Configuration, if provided, should be a
+dictionary.
```

Comment:
Combine intro P
```
The list of `plots` contains one or more user-defined top-level plots (paths
relative to the location of `dvc.yaml`).
Every plot needs a unique file path as identifier.
Optional configuration can be given as a dictionary.
```

Reply:
> Every plot needs a unique file path as identifier.

You can have many different plots all based on the same file.

Reply:
> different plots all based on the same file

I see. This "path vs ID" situation is a bit confusing. Explanations could improve on that I think... ⌛